Sources/ArcGISToolkit/Utility/RepresentedUITextView.swift

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+/// A represented `UITextView`.
+///
+/// A `UITextView` has noticeably better performance over a SwiftUI `TextField` when dealing with
+/// large input (e.g. thousands of characters).
+struct RepresentedUITextView: UIViewRepresentable {
+    @Binding var text: String
+    
+    func makeUIView(context: Context) -> UITextView {
+        let uiTextView = UITextView()
+        uiTextView.delegate = context.coordinator
+        return uiTextView
+    }
+    
+    func updateUIView(_ uiTextView: UITextView, context: Context) {
+        uiTextView.text = text
+    }
+    
+    func makeCoordinator() -> Coordinator {
+        Coordinator(text: $text)
+    }
+    
+    class Coordinator: NSObject, UITextViewDelegate {
+        var text: Binding<String>
+        
+        init(text: Binding<String>) {
+            self.text = text
+        }
+        
+        func textViewDidChange(_ textView: UITextView) {
+            text.wrappedValue = textView.text
```

Comment:
Performance improves even more if we only update the bound value via `textViewDidEndEditing` but then we lose the live footer character count and validation updates.

We could look at reworking how `InputFooter` works but I'm not sure if this'll be a common enough problem to do that at this point.